src/mds/topology/topology_service_manager.cpp

```diff
@@ -1458,6 +1488,15 @@ void TopologyServiceManager::ListUnAvailCopySets(
     response->set_statuscode(kTopoErrCodeSuccess);
 }
 
+void TopologyServiceManager::ConvertCopyset(const CopySetInfo& in,
```

Comment:
Can use protobuf's 'CopyFrom'.

Reply:
> Can use protobuf's 'CopyFrom'.

But the type of param `in` isn't a proto type